htlcswitch/link.go

```diff
@@ -2144,6 +2149,14 @@ func (l *channelLink) ChanID() lnwire.ChannelID {
 	return lnwire.NewChanIDFromOutPoint(l.channel.ChannelPoint())
 }
 
+// IsDisabled returns the ForwardingPolicy Disabled status for the
+// channel link.
+//
+// NOTE: Part of the ChannelLink interface.
+func (l *channelLink) IsDisabled() bool {
```

Comment:
Not sure we need to extend this interface, given this can likely be hidden behind `CheckHtlcForward` (to ensure we don't forward to a disabled channel), and `CheckHtlcTransit` to ensure we don't attempt to send a payment out from a disabled channel.